src/runtime_src/core/common/config_reader.h

```diff
@@ -705,6 +705,17 @@ get_flag_kds_sw_emu()
   return value;
 }
 
+// This flag is added to support force xclbin download eventhough same xclbin is already programmed.
+// This is required for aie reset/reinit in next run. Aie is not clean after first
+// run. We need to work with aie team to figureout a solution to reset/reinit AIE in second run.
+// This flag is used only in edge flow
```

Comment:
Can we make it work for DC as well?

Reply:
Sure @maxzhen . We can make changes in DC as well, I haven't made as we don't have any use case today.

Reply:
I think that it is good to keep it consistent across Edge and DC since xrt.ini is common file. This also might be a potential solution for CR-1089930.

Reply:
Sure Max. I will make the change.

Reply:
@maxzhen , Made changes in DC flow as well (Only in Linux)
@arpitxilinx , We can make similar changes to Windows. (Will file a separate PR for it)